TeachingRecordSystem/tests/TeachingRecordSystem.SupportUi.Tests/PageTests/Cases/EditCase/IndexTests.cs

```diff
@@ -0,0 +1,159 @@
+using TeachingRecordSystem.Core.Dqt.Models;
+
+namespace TeachingRecordSystem.SupportUi.Tests.PageTests.Cases.EditCase;
+
+[Collection(nameof(DisableParallelization))]
+public class IndexTests : TestBase
+{
+    public IndexTests(HostFixture hostFixture)
+        : base(hostFixture)
+    {
+        XrmFakedContext.DeleteAllEntities<Incident>();
```

Comment:
Do we need this? (If not `[Collection(nameof(DisableParallelization))]` can go too.)

Reply:
Removed